src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.tsx

```diff
@@ -75,21 +73,18 @@ export const Report = ({ activity, identifiers = [], versions = [] }: ReportProp
 
     setLatestReportError(null);
     try {
-      const { data } = await getLatestReport({
+      const res = await getLatestReport({
```

Comment:
pls change variable name for more understandable
```suggestion
      const latestReport = await getLatestReport({
```

Reply:
done